cms/static/js/views/video_transcripts.js

```diff
@@ -0,0 +1,101 @@
+define(
+    ['underscore', 'gettext', 'js/views/baseview', 'edx-ui-toolkit/js/utils/html-utils',
+        'edx-ui-toolkit/js/utils/string-utils', 'text!templates/video-transcripts.underscore'],
+    function(_, gettext, BaseView, HtmlUtils, StringUtils, videoTranscriptsTemplate) {
+        'use strict';
+
+        var TRANSCRIPT_DOWNLOAD_FILE_FORMAT = 'srt',
+            VideoTranscriptsView;
+
+        VideoTranscriptsView = BaseView.extend({
+            tagName: 'div',
+
+            events: {
+                'click .toggle-show-transcripts-button': 'toggleShowTranscripts'
+            },
+
+            initialize: function(options) {
+                this.transcripts = options.transcripts;
+                this.edxVideoID = options.edxVideoID;
+                this.clientVideoID = options.clientVideoID;
+                this.transcriptAvailableLanguages = options.transcriptAvailableLanguages;
+                this.videoSupportedFileFormats = options.videoSupportedFileFormats;
+                this.template = HtmlUtils.template(videoTranscriptsTemplate);
+            },
+
+            /*
+            Sorts object by value and returns a sorted array.
+            */
+            sortByValue: function(itemObject) {
+                var sortedArray = [];
+                _.each(itemObject, function(value, key) {
+                    // Push each JSON Object entry in array by [value, key]
+                    sortedArray.push([value, key]);
+                });
+                return sortedArray.sort();
+            },
+
+            /*
+            Returns transcript title.
+            */
+            getTranscriptClientTitle: function() {
+                var clientTitle = this.clientVideoID;
+                // Remove video file extension for transcript title.
+                _.each(this.videoSupportedFileFormats, function(videoFormat) {
```

Comment:
if we are going to separate column for transcripts then we won't be needing this filename (showing up)

Reply:
For this MVP, we would need file name. Transcript status UX has not been decided yet so I think we should proceed with this for now.

Reply:
okay.